File: LeetCode-Problems/Easy/Excel-Sheet-Column-Title.py
# Python
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        number = ''
        x = n
        rStr = ""
        while (x > 0):
            number = chr(((x-1) % 26)+ 65)
            rStr += number
            x = (x-1) / 26
        return rStr[::-1]

File: LeetCode-Problems/Easy/Excel-Sheet-Column-Title.java
//Java
import java.lang.StringBuilder;
class Solution {
    public String convertToTitle(int n) {
        char ascii;
        StringBuilder rStr = new StringBuilder();
        int x = n;
        while (x > 0){
            ascii = (char) (((x-1) % 26) + 65);
            rStr.insert(0, ascii);
            x = (x-1) / 26;
        }
        return rStr.toString();
    }
}
